pkg/validate/serialize.go

```diff
@@ -13,38 +13,41 @@ import (
 
 // ValidateCSVManifest takes in name of the yaml file to be validated, reads
 // it, and calls the unmarshal function on rawYaml.
-func ValidateCSVManifest(yamlFileName string) validator.Error {
+func ValidateCSVManifest(yamlFileName string) (manifestResult validator.ManifestResult) {
 	rawYaml, err := ioutil.ReadFile(yamlFileName)
 	if err != nil {
-		return validator.IOError(fmt.Sprintf("Error in reading %s file:   #%s ", yamlFileName, err), yamlFileName)
+		manifestResult.Errors = append(manifestResult.Errors, validator.IOError(fmt.Sprintf("Error in reading %s file:   #%s ", yamlFileName, err), yamlFileName))
+		return
 	}
 
 	// Value returned is a marshaled go type (CSV Struct).
 	csv, err := unmarshal(rawYaml)
 	if err != nil {
-		return validator.InvalidParse(fmt.Sprintf("Error unmarshalling YAML to OLM's csv type for %s file:  #%s ", yamlFileName, err), yamlFileName)
+		manifestResult.Errors = append(manifestResult.Errors, validator.InvalidParse(fmt.Sprintf("Error unmarshalling YAML to OLM's csv type for %s file:  #%s ", yamlFileName, err), yamlFileName))
+		return
 	}
 
 	v := &CSVValidator{}
 	if err := v.AddObjects(csv); err != (validator.Error{}) {
-		return err // TODO: update when 'AddObjects' returns an actual error.
+		manifestResult.Errors = append(manifestResult.Errors, err)
+		return // TODO: update when 'AddObjects' returns an actual error.
 	}
-	fmt.Println("Running", v.Name())
+	fmt.Println("\nRunning", v.Name())
 	for _, errorLog := range v.Validate() {
 		fmt.Println("Validating CSV", errorLog.Name)
```

Comment:
```suggestion
		fmt.Printf("Validating CSV %s\n\n", errorLog.Name)
```
and remove `fmt.Println()` below.